lib/dnsimple/struct/certificate.rb

```diff
@@ -0,0 +1,40 @@
+module Dnsimple
+  module Struct
+
+    class Certificate < Base
+      # @return [Fixnum] The certificate ID in DNSimple.
+      attr_accessor :id
+
+      # @return [Fixnum] The associated domain ID.
+      attr_accessor :domain_id
+
+      # @return [String] The certificate name.
+      attr_accessor :name
```

Comment:
name is an internal attribute that I'd like to eventually remove. I'd prefer to not include it, as we already provide `common_name`.